code/modules/reagents/reagent_containers/drugs.dm

```diff
@@ -12,80 +12,81 @@
 
 	var/label_text = ""
 
-	New()
```

Comment:
this whole thing is shit. why wasn't this made a subtype of glass with different fillings/updat_icons. ughhhhh